src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -398,4 +411,89 @@ class TableView extends React.Component<TableViewProps> {
   }
 }
 
+const UNSEARCHABLE_FIELDS = ['p99', 'p75', 'p95', 'last_seen'];
+
+const ExpandAggregateRow = (props: {
+  children: React.ReactNode;
+  eventView: EventView;
+  column: TableColumn<keyof TableDataRow>;
+  dataRow: TableDataRow;
+  location: Location;
+}) => {
+  const {children, column, dataRow, eventView, location} = props;
+
+  const {eventViewField} = column;
+
+  const exploded = explodeField(eventViewField);
+  const {aggregation} = exploded;
+
+  if (aggregation === 'count') {
```

Comment:
What about `count_unique`, `min`, `max` etc?

Reply:
@markstory `count_unique` is somewhat ruled out; since we cannot do "distinct" search on the search conditions.

I think for this prototype, we can just focus on `count()`.